reactioncommerce/reaction
# (feat): remove ability for non-marketplace owner to switch shops

This update changes the shop selector dropdown to only allow marketplace owners to switch shops. If you are a single shop owner, the selector will no longer be displayed, and you will no longer be allowed to switch active shops.

To test:
- Login as Reaction admin
- Invite new shop owner
- Make new shop active
- See you can switch back and forth between shops as Marketplace owner
- Make sure the last shop you land on is the original Marketplace shop (this will test to make sure the setting isn't saved in Browser storage)

THEN:

- Login as new shop
- See shop selector is not displayed, AND you are able to make all admin actions on your shop (i.e. the active shop is your shop, not the marketplace shop)
